### ISSUE TITLE
[Feature] Community Message

### ISSUE THREAD

**Issue by shiipou** [Chocobozzz/PeerTube#2549]
It can be a cool feature to add a possibility to post some message with picture (or other small media) to user channel.

This can help to inform of anything to our community. I think this can be useful for channel owner and viewer. (It can also be used by instance owner to post server changelog)

Like this picture I made : 

![Adobe_Post_20200309_1035040 42410951983178524](https://user-images.githubusercontent.com/38187238/76200399-dcf0c280-61f1-11ea-9e4b-7a5eda414fc1.png)

**Comment by rigelk** [comment]
I don't think such messages have their place in the video listings. They should have a dedicated place: a new view, or a column in the subscriptions. Channel-wise, a "Community" nav-item and view could be a way to do it.

**Comment by shiipou** [comment]
Yes it will be better in a specific view. but it need to be in the same as video that viewer follow because if not, nobody will really view it.

Channel-Wize or all followed channel-wide timeline can be great.

**Comment by 3risian** [comment]
I personally don't think announcements should be added as a fake video placed among the others. (If an admin really wants that, I would prefer it pinned/stickied so it stays as the first video, and highlighted in a way that makes it clear it isn't a normal video)

I think a dedicated place is best, with some obvious notification on the homepage/viewpage listing the title and linking them there. I know a lot of sites add a small highlighted row at the very top of the page for important announcements or notifications, with an 'X' button for closing it. I feel like one of those would be good. Bonus if it becomes less highlighted over time, so users will notice new updates instead of subconsciously ignoring them.

**Comment by Chocobozzz** [comment]
Merging this issue in https://github.com/Chocobozzz/PeerTube/issues/3997
